docs/dev/guidelines/client-design.rst

```diff
@@ -0,0 +1,241 @@
+*************
+Client Theming
+*************
```

Comment:
```suggestion
**************
Client Theming
**************
```
Sphinx complains about the stars not matching the text length.

Reply:
👍